config/filament-authentication-log.php

```diff
@@ -10,6 +10,10 @@
         \App\Models\User::class,
     ],
 
+    'authenticatable' => [
+        'field-to-display' => 'name',
```

Comment:
Hi @abbasmashaddy72, thank you for working on on this! :)
Could you change the default config to `null` so it will not break if someone doesn't have the field `name` on authenticatable table:

```php
'authenticatable' => [
        'field-to-display' => null,
],  
```

Reply:
Done